docs/concepts/models/python_models.md

```diff
@@ -90,7 +90,40 @@ Optional pre/post-statements allow you to execute SQL commands before and after
 
 For example, pre/post-statements might modify settings or create indexes. However, be careful not to run any statement that could conflict with the execution of another statement if models run concurrently, such as creating a physical table.
 
-Pre- and post-statements are issued with the SQLMesh [`fetchdf` method](../../reference/cli.md#fetchdf) [described above](#execution-context).
+You can set the `pre_statements` and `post_statements` arguments to a list of SQL strings and/or SQLGlot expressions to define the model's pre/post-statements.
+
+``` python linenums="1" hl_lines="6-7"
+def execute(
```

Comment:
Include @model decorator in example

Reply:
right updated this good catch it was in the wrong arguments before